From b86fdd54f74c4d4540fd5d5c102673df965ba251 Mon Sep 17 00:00:00 2001
From: MilhouseVH <milhouseVH.github@nmacleod.com>
Date: Sun, 08 Oct 2017 23:50:38 +0100
Subject: [PATCH] default is optimized

Refreshed MilhouseVH patch from Python3.7. Original message:

Do *not* enable Py_OptimizeFlag=2 (or higher) as this will stop
__doc__ output from being generated which will prevent the qemu
package for Generic from building.

---
diff --git a/Python/initconfig.c b/Python/initconfig.c
index a7cd08d..7ea28fa 100644
--- a/Python/initconfig.c
+++ b/Python/initconfig.c
@@ -619,7 +619,7 @@ int Py_VerboseFlag = 0; /* Needed by import.c */
 int Py_QuietFlag = 0; /* Needed by sysmodule.c */
 int Py_InteractiveFlag = 0; /* Previously, was used by Py_FdIsInteractive() */
 int Py_InspectFlag = 0; /* Needed to determine whether to exit at SystemExit */
-int Py_OptimizeFlag = 0; /* Needed by compile.c */
+int Py_OptimizeFlag = 1; /* Needed by compile.c */
 int Py_NoSiteFlag = 0; /* Suppress 'import site' */
 int Py_BytesWarningFlag = 0; /* Warn on str(bytes) and str(buffer) */
 int Py_FrozenFlag = 0; /* Needed by getpath.c */
-- 
2.53.0

